tools/gprof2dot.py

```diff
@@ -35,24 +35,23 @@
 # Python 2.x/3.x compatibility
 if sys.version_info[0] >= 3:
```

Comment:
this is a 3rd party program. I'm not even sure this is hooked up to any of the other tools in here

Reply:
Yea, in hindsight maybe I should have just had the linter ignore this file, but as it stands this change will make it pass the linter and it should be Python 3 ready.

If this is not used by anything, should it maybe be removed from the repository?

Reply:
yeah, I've taken some notes of reviewing whether all the log parsers still need to be here